foundation-ui/utils/ResourceTableUtil.js

```diff
@@ -45,6 +46,8 @@ var ResourceTableUtil = {
 
   getSortFunction(tieBreakerProp) {
```

Comment:
How about a small refactoring? The name of the file is `ResourceTableUtil` but `ResourceTable` does't exist. That's ok, the `getSortFunction` is being used only by the `TaskTable` if I'm not mistaken. Let's extract this function into `TaskTableUtil`. Then I won't be doing a hardcore soul searching justifying presence of a specific constant `TaskStatusSorting` inside a generic function `getSortFunction`. Does it make sense? 

Happy to discuss this today.